js/src/setup-mc/setup-stepper/setup-accounts/google-mc-account/disabled-card.js

```diff
@@ -14,13 +14,9 @@ const DisabledCard = () => {
 		<AccountCard
 			disabled
 			appearance={ APPEARANCE.GOOGLE_MERCHANT_CENTER }
-			description={ __(
-				'Required to sync products and list on Google',
-				'google-listings-and-ads'
-			) }
 			indicator={
 				<Button isSecondary disabled>
-					{ __( 'Connect', 'google-listings-and-ads' ) }
+					{ __( 'Create Account', 'google-listings-and-ads' ) }
```

Comment:
When checking the implementations against visual design, I found this button text displays "Create Account". So I adjusted it by the way.